spec/grom/reader_spec.rb

```diff
@@ -48,5 +49,22 @@
 
       expect(reader.instance_variable_get(:@edges_by_subject).size).to eq(14)
     end
+
+    context 'article data' do
```

Comment:
Could this section have a test which states what explicitly you are testing?

It is not clear that this is to test blank node linking